covid_moonshot/analysis/plots.py

```diff
@@ -251,6 +253,67 @@ def plot_convergence(
     return fig
 
 
+def plot_poor_convergence_fe_table(
+    runs: List[Run], energy_cutoff_kcal: float = 1.0,
+) -> plt.Figure:
+    """
+    Plot table of poorly converging free energy estimates with GEN
+
+    Parameters
+    ----------
+    runs : list of Run
+        Relative free energies (in kT)
+    energy_cutoff_kcal : float
+        Cutoff to consider a result as poorly converged (in kcal/mol)
+
+    Returns
+    -------
+    Figure
+        Figure containing the plot
+
+    """
+
+    complex_phases = [run.analysis.complex_phase for run in runs]
+    job_ids = [run.details.JOBID for run in runs]
+
+    std_dev_store = []
+    jobid_store = []
+
+    for run in runs:
+
+        try:
+
+            complex_gens = run.analysis.complex_phase.gens
+            std_dev = np.std([gen.free_energy.delta_f for gen in complex_gens])
+
+            if std_dev * KT_KCALMOL >= energy_cutoff_kcal:
+
+                jobid_store.append(run.details.JOBID)  # JOBID X should = RUN X
+                std_dev_store.append(np.round(std_dev * KT_KCALMOL, 3))
+
+        except AttributeError:  # skip if no delta_f recorded
+            continue
+
+    # Create 2D list for table input
+    data = [[i, j] for i, j in zip(jobid_store, std_dev_store)]
+
```

Comment:
I ran this on a subset of the data and got `IndexError: list index out of range` from `table`. I think we'll want to add something like
```python
if not data:
    return None
```
here (and change return type to `Optional[plt.Figure]`)